paasta_tools/autoscaling_lib.py

```diff
@@ -344,6 +344,16 @@ def humanize_error(error):
         return 'utilization within thresholds'
 
 
+def is_task_healthy(task):
+    """
+    Check a marathon task has healthcheck results
+    and check that they are all healthy.
+    """
+    if task.health_check_results:
+        return all([hcr.alive for hcr in task.health_check_results])
```

Comment:
Yea if there are no healthchecks then we have to use *something* to make a heuristic about a task being healthy. 1m is better than nothing, basically we want to make sure that constantly churning tasks are not considered "healthy"